hugegraph-api/src/main/java/org/apache/hugegraph/api/cypher/CypherClient.java

```diff
@@ -136,10 +144,9 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        final StringBuffer sb = new StringBuffer("CypherClient{");
-        sb.append("userName='").append(userName).append('\'');
-        sb.append(", token='").append(token).append('\'');
-        sb.append('}');
+        final StringBuilder sb = new StringBuilder("CypherClient{");
+        sb.append("userName='").append(userName).append('\'')
+          .append(", token='").append(token).append('\'').append('}');
```

Comment:
the style checker requires at least 4 spaces indentation, please rename sb to builder